tests/flow/test_concurrent_query.py

```
@@ -0,0 +1,422 @@
+
+GRAPH_ID = "concurrent_query"       # Graph identifier.
+SECONDARY_GRAPH_ID = GRAPH_ID + "2" # Secondery graph identifier.
+CLIENT_COUNT = 16                   # Number of concurrent connections.
+people = ["Roi", "Alon", "Ailon", "Boaz", "Tal", "Omri", "Ori"]
+
+async def delete_graph(g):
+    # Try to delete graph.
+    try:
+        await g.delete()
+        return True
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)